### PR TITLE
Bigtable: Emulator fixes

### PR DESCRIPTION
**Pull request by sumit-ql**
This is fix for https://github.com/googleapis/google-cloud-python/issues/6287

### CODE REVIEW

**Comment by sumit-ql** [comment]
@sduskis @tseaver Please review.

File: bigtable/tests/unit/test_client.py [hunk 1]
```diff
@@ -86,6 +212,19 @@ def test_constructor_both_admin_and_read_only(self):
...
+        credentials = _make_credentials()
+        emulator_host = "localhost:8081"
+        with mock.patch('os.getenv') as mocked:
+            mocked.return_value = emulator_host
+            client = self._make_one(project=self.PROJECT,
+                                    credentials=credentials)
+
+        self.assertEqual(client._emulator_host, emulator_host)
+        self.assertIsInstance(client._emulator_channel,
+                              grpc._channel.Channel)
```
Comment (tseaver) [inline bigtable/tests/unit/test_client.py]: This really wants an assertion that `emulator_host` was correctly passed into the `grpc.insecure_channel` factory, which we could mock out via:

```python
    def test_constructor_with_emulator_host(self):
        import grpc
        credentials = _make_credentials()
        emulator_host = "localhost:8081"
        with mock.patch('os.getenv') as mocked:
            with mock.patch('grpc.insecure_channel') as factory:
                mocked.return_value = emulator_host
                client = self._make_one(
                    project=self.PROJECT, credentials=credentials)

        self.assertEqual(client._emulator_host, emulator_host)
        self.assertIsInstance(client._emulator_channel,
                              grpc._channel.Channel)
        factory.assert_called_once_with(emulator_host)
```

**Comment by sduskis** [comment]
@sumit-ql, what's your timeline for working on this?  This is blocking our next release.

File: bigtable/tests/unit/test_client.py (hunk 1 above)
Comment (tseaver) [reply bigtable/tests/unit/test_client.py]: 6b0abee04bd8a043025c8b1ba116a3bf80d98d49

File: bigtable/google/cloud/bigtable/client.py [hunk 2]
```diff
@@ -58,6 +61,61 @@
...
+    :returns: A BigtableInstanceAdmin instance.
+    """
+    if client._emulator_host is None:
+        return bigtable_admin_v2.BigtableInstanceAdminClient(
+            credentials=client._credentials,
+            client_info=_CLIENT_INFO)
+    else:
+        return bigtable_admin_v2.BigtableInstanceAdminClient(
+            channel=client._emulator_channel,
+            client_info=_CLIENT_INFO)
```
[the remark this answers is not in the document]
Comment (tseaver) [reply bigtable/google/cloud/bigtable/client.py]: fe70cbb667bc73e1761b5913597d1686389c2953

**Pull request merged**